src/middlewares/checkUser.js

```diff
@@ -0,0 +1,35 @@
+import models from '../api/models';
+import status from '../helpers/constants/status.codes';
+
+const { User } = models;
+/**
+ * check if user exist
+ *
+ * @export
+ * @class CheckUser
+ */
+class CheckUser {
+  /**
+   * this is a middleware which checks if the user is registered in to our database.
```

Comment:
We know its a middleware, just explain the inner workings of the function  @Ramadhan0

Reply:
I am checking if a user to be blocked exists, we can not block a user who doesn't exist